webpack/scenes/Subscriptions/SubscriptionsTableSchema.js

```diff
@@ -1,14 +1,53 @@
 import React from 'react';
-import { Icon } from 'patternfly-react';
+import { Icon, Table } from 'patternfly-react';
 import helpers from '../../move_to_foreman/common/helpers';
+import { entitlementsInlineEditFormatter } from './EntitlementsInlineEditFormatter';
 import { headerFormat, cellFormat } from '../../move_to_foreman/components/common/table';
 
-export const columns = [
+export const columns = (inlineEditController, selectionController) => [
+  {
+    property: 'select',
+    header: {
+      label: __('Select all rows'),
+      props: {
+        index: 0,
+      },
+      formatters: [
+        label => (
+          <Table.SelectionHeading aria-label={label}>
```

Comment:
Could you factor it out into a reusable formatter? I guess this is a pattern that will be repeated often.
Ideally we should use it on the upstream subscriptions page too.

Reply:
:+1: